qa/rpc-tests/cashlibtest.py

```diff
@@ -31,7 +31,7 @@ def setup_network(self, split=False):
         self.sync_all()
 
     def run_test(self):
-
+        FEE = 0
```

Comment:
what's the purpose of this variable? Is used in the rest of the test to be subtracted to something else but since is set to 0...

guess that a comment to explain could be useful

Reply:
It lets you create tx with different fees.  But a 0 fee tx is being accepted now so no need in the normal qa test to include a fee.